src/main/java/org/graylog/integrations/notifications/types/SlackEventNotificationConfig.java

```diff
@@ -125,10 +122,6 @@ public ValidationResult validate() {
         ValidationResult validation = new ValidationResult();
         final Matcher matcher = pattern.matcher(webhookUrl());
 
-        if (backlogSize() < 0 ) {
-            validation.addError(FIELD_BACKLOG_SIZE, INVALID_BACKLOG_ERROR_MESSAGE);
-        }
```

Comment:
This check should remain.  Without it, users can send a test notification with `backlog_size` of -1.